website/src/app/page.tsx

```diff
@@ -1,13 +1,29 @@
 "use client";
 
 import BannerComponent from "@/components/banner/Banner";
+import DetailComponent from "@/components/details/Details";
 import HeaderComponent from "@/layout/header/Header";
+import { reverse } from "dns";
 
 export default function Home() {
+  const Details = {
+     reverse:[
```

Comment:
const Details = [{
   imageAlignment :  left,
   image : "/images/About.png",
  title: "Detail",
  subTitle:"Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum, ad!",
  description:"Lorem ipsum dolor sit amet consectetur, adipisicing elit. Adipisci odit, in inventore obcaecati, vitae id provident, nihil vero voluptas eius eveniet veritatis reprehenderit rerum. Dicta reiciendis maiores quos explicabo nisi!",
  button :{
  text: 'click me',
  onClick : () => {}
}
},
{
   imageAlignment :  right,
   image : "/images/About.png",
  title: "Detail",
  subTitle:"Lorem ipsum dolor sit amet consectetur, adipisicing elit. Eum, ad!",
  description:"Lorem ipsum dolor sit amet consectetur, adipisicing elit. Adipisci odit, in inventore obcaecati, vitae id provident, nihil vero voluptas eius eveniet veritatis reprehenderit rerum. Dicta reiciendis maiores quos explicabo nisi!",
  button :{
  text: 'click me',
  onClick : () => {}
}
}
]
  add this kind of configuration